pkg/services/ngalert/notifier/multiorg_alertmanager.go

```diff
@@ -142,8 +142,12 @@ func NewMultiOrgAlertmanager(
 		peer:           &NilPeer{},
 	}
 
-	if err := moa.setupClustering(cfg); err != nil {
-		return nil, err
+	if cfg.UnifiedAlerting.SkipClustering {
+		l.Debug("Skipping setting up clustering for MOA")
```

Comment:
I'd make this Info - it's low volume, only getting logged once startup, and could be a useful indicator of something working unexpectedly (e.g. we skip clustering when we actually want it).

```suggestion
		l.Info("Skipping setting up clustering for MOA")
```